{"name": "test_add[except]", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == '请输入数字'\n  +None\n  -'请输入数字'", "trace": "self = <test_cal_new.TestCalc object at 0x03D45310>, get_calc = <pythoncode.calculator.Calculator object at 0x03D45030>, get_add_datas = ['', 5, '请输入数字']\n\n    @pytest.mark.run(order=1)\n    def test_add(self, get_calc, get_add_datas):\n        result = None\n        try:\n            result = get_calc.add(get_add_datas[0], get_add_datas[1])\n            if isinstance(result, float):\n                result = round(result, 2)\n        except Exception as e:\n            print(e)\n>       assert result == get_add_datas[2]\nE       AssertionError: assert None == '请输入数字'\nE         +None\nE         -'请输入数字'\n\ntest_cal_new.py:20: AssertionError"}, "parameters": [{"name": "get_add_datas", "value": "['', 5, '请输入数字']"}], "start": 1609672620504, "stop": 1609672620504, "uuid": "5703a580-ca4d-417c-8c70-b113f0c97240", "historyId": "1b83f4905575390cd67cc1b92e20f997", "testCaseId": "c66a2a94fce4eee5f6d73963586d32c0", "fullName": "test_cal_new.TestCalc#test_add", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_cal_new"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-IOCKJLK"}, {"name": "thread", "value": "13264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cal_new"}]}